parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -429,6 +432,17 @@ should_skip_nvidia_drivers() {
     echo "$should_skip"
 }
 
+should_disable_kubelet_serving_certificate_rotation() {
```

Comment:
function name format doesn't match the below functionName. should it be 

`shouldDisableKubeletServingCertificateRotation()` instead ? of the other one is wrong ?

Reply:
I decided to copy the format used for the skip gpu driver function, I think we should probably align on a style guide eventually...